data/aliases.js

```diff
@@ -826,29 +826,801 @@ let BattleAliases = {
 	"zwater": "Hydro Vortex",
 
 	// Japanese names
-	"birijion": "Virizion",
-	"terakion": "Terrakion",
-	"agirudaa": "Accelgor",
-	"randorosu": "Landorus",
-	"urugamosu": "Volcarona",
-	"erufuun": "Whimsicott",
+	"fushigidane": "Bulbasaur",
+	"fushigisou": "Ivysaur",
+	"fushigibana": "Venusaur",
+	"hitokage": "Charmander",
+	"rizaado": "Charmeleon",
+	"rizaadon": "Charizard",
+	"zenigame": "Squirtle",
+	"kameiru": "Wartortle",
+	"kamekkusu": "Blastoise",
+	"kyatapi": "Caterpie",
+	"toranseru": "Metapod",
+	"batafuri": "Butterfree",
+	"bidoru": "Weedle",
+	"kokuun": "Kakuna",
+	"supiaa": "Beedrill",
+	"poppo": "Pidgey",
+	"pijon": "Pidgeotto",
+	"pijotto": "Pidgeot",
+	"koratta": "Rattata",
+	"ratta": "Raticate",
+	"onisuzume": "Spearow",
+	"onidoriru": "Fearow",
+	"bo": "Ekans",
+	"bokku": "Arbok",
+	"pikachuu": "Pikachu",
+	"raichuu": "Raichu",
+	"sando": "Sandshrew",
+	"sandopan": "Sandslash",
+	"nidoranmesu": "Nidoran♀",
+	"nidokuin": "Nidoqueen",
+	"nidoranosu": "Nidoran♂",
+	"nidokingu": "Nidoking",
+	"pippi": "Clefairy",
+	"pikushi": "Clefable",
+	"rokon": "Vulpix",
+	"kyuukon": "Ninetales",
+	"purin": "Jigglypuff",
+	"pukurin": "Wigglytuff",
+	"zubatto": "Zubat",
+	"gorubatto": "Golbat",
+	"nazonokusa": "Oddish",
+	"kusaihana": "Gloom",
+	"rafureshia": "Vileplume",
+	"parasu": "Paras",
+	"parasekuto": "Parasect",
+	"konpan": "Venonat",
+	"morufon": "Venomoth",
+	"diguda": "Diglett",
+	"dagutorio": "Dugtrio",
+	"nyaasu": "Meowth",
+	"perushian": "Persian",
+	"kodakku": "Psyduck",
+	"gorudakku": "Golduck",
+	"manki": "Mankey",
+	"okorizaru": "Primeape",
+	"gaadi": "Growlithe",
+	"uindi": "Arcanine",
+	"nyoromo": "Poliwag",
+	"nyorozo": "Poliwhirl",
+	"nyorobon": "Poliwrath",
+	"keishi": "Abra",
+	"yungeraa": "Kadabra",
+	"fuudin": "Alakazam",
+	"wanriki": "Machop",
+	"gouriki": "Machoke",
+	"kairiki": "Machamp",
+	"madatsubomi": "Bellsprout",
+	"utsudon": "Weepinbell",
+	"utsubotto": "Victreebel",
+	"menokurage": "Tentacool",
+	"dokukurage": "Tentacruel",
+	"ishitsubute": "Geodude",
+	"goroun": "Graveler",
+	"gorounya": "Golem",
+	"ponita": "Ponyta",
+	"gyaroppu": "Rapidash",
+	"yadon": "Slowpoke",
+	"yadoran": "Slowbro",
+	"koiru": "Magnemite",
+	"reakoiru": "Magneton",
+	"kamonegi": "Farfetch'd",
+	"doudou": "Doduo",
+	"doudorio": "Dodrio",
+	"pauwau": "Seel",
+	"jugon": "Dewgong",
+	"betobetaa": "Grimer",
+	"betobeton": "Muk",
+	"sherudaa": "Shellder",
+	"parushen": "Cloyster",
+	"gousu": "Gastly",
+	"gousuto": "Haunter",
+	"gengaa": "Gengar",
+	"iwaaku": "Onix",
+	"suripu": "Drowzee",
```

Comment:
Drowzee is `suriipu` in the usual romanizations. `surīpu` is Modern Hepburn but none of your other romanizations are Modern Hepburn, which makes this one suspicious.